project.mk

```diff
@@ -6,4 +6,7 @@ GIT_ROOT?=$(shell git rev-parse --show-toplevel 2>&1)
 # WARNING: REPO_NAME will default to the current directory if there are no remotes
 REPO_NAME?=$(shell basename $$((git config --get-regex remote\.*\.url 2>/dev/null | cut -d ' ' -f2 || pwd) | head -n1 | sed 's|.git||g'))
 
-SELECTOR_SYNC_SET_DESTINATION?=${GIT_ROOT}/hack/00-osd-${REPO_NAME}.selectorsyncset.yaml.tmpl
\ No newline at end of file
+SELECTOR_SYNC_SET_DESTINATION?=${GIT_ROOT}/hack/00-osd-${REPO_NAME}.selectorsyncset.yaml.tmpl
+
+SOURCE_CODE_HASH ?= $(shell (cat monitor/* 2>/dev/null || echo "no-source-code") | openssl dgst -sha256)
```

Comment:
Wouldn't it be the case of doing here: `openssl dgst -sha256 -binary | xxd -p`, to avoid the leading `(stdin)` text?

Reply:
@rogbas check the updated file, let me know what you think of the workaround?

Reply:
+1 for the filter with grep, but I didn't understand the double call to `openssl -dgst -sha256`

`openssl dgst -sha256 | openssl dgst -sha256 | grep -o '[[:xdigit:]][[:xdigit:]]*$'`

Reply:
the double call is a defect.

Reply:
Should be fixed now in 3d6fd0c